sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/changestreams/SpannerChangeStreamErrorTest.java

```diff
@@ -0,0 +1,481 @@
+        StatementResult.query(getPartitionsAfterStatement2, getPartitionResultSet));
+
+    Statement changeStreamQueryStatement =
+        Statement.newBuilder(
+                "SELECT * FROM READ_my-change-stream(   start_timestamp => @startTimestamp,   end_timestamp => @endTimestamp,   partition_token => @partitionToken,   read_options => null,   heartbeat_milliseconds => @heartbeatMillis)")
+            .bind("startTimestamp")
+            .to(now)
+            .bind("endTimestamp")
+            .to(after3Seconds)
+            .bind("partitionToken")
+            .to((String) null)
+            .bind("heartbeatMillis")
+            .to(500)
+            .build();
+    ResultSetMetadata readChangeStreamResultSetMetadata =
+        ResultSetMetadata.newBuilder()
+            .setRowType(
+                StructType.newBuilder()
+                    .addFields(
+                        Field.newBuilder()
+                            .setName("COL1")
+                            .setType(
+                                Type.newBuilder()
+                                    .setCode(TypeCode.ARRAY)
+                                    .setArrayElementType(
+                                        Type.newBuilder()
+                                            .setCode(TypeCode.STRUCT)
+                                            .setStructType(
+                                                StructType.newBuilder()
+                                                    .addFields(
+                                                        Field.newBuilder()
+                                                            .setName("field_name")
+                                                            .setType(
+                                                                Type.newBuilder()
+                                                                    .setCode(TypeCode.STRUCT)
+                                                                    .setStructType(
+                                                                        StructType.newBuilder()
+                                                                            .addFields(
+                                                                                Field.newBuilder()
+                                                                                    .setType(
+                                                                                        Type
+                                                                                            .newBuilder()
+                                                                                            .setCode(
+                                                                                                TypeCode
+                                                                                                    .STRING))
+                                                                                    .build())
```

Comment:
nit: I think you can remove the builds (but the most outer one) and it should still compile.